content/millennium/r4/financial/charge-item.md

```diff
@@ -0,0 +1,111 @@
+---
+title: ChargeItem | R4 API
+---
+
+# ChargeItem
+
+* TOC
+{:toc}
+
+## Overview
+
+The resource ChargeItem describes the provision of healthcare provider products for a certain patient, therefore referring not only to the product, but containing in addition details of the provision, like date, time, amounts and participating organizations and persons.
+Main Usage of the ChargeItem is to enable the billing process and internal cost allocation.
+
+The following fields are returned if valued:
+
+* [ChargeItem id](https://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Status](https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.status){:target="_blank"}
+* [Code](https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.code){:target="_blank"}
+* [Subject](https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.subject){:target="_blank"}
+* [Context](https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.context){:target="_blank"}
+* [occurrenceDateTime](https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.occurrence_x_){:target="_blank"}
```

Comment:
Should this be capitalized and spaced to Occurrence Date Time, or should we just call it Occurrence?

Reply:
I updated it to Occurrence [here](https://github.com/cerner/fhir.cerner.com/pull/605/commits/64ddf564c76cd6938adb0bdff00feda5cff09c86#diff-16d4f8f74626c62d57a8b71d6be699f60e72fffed8636bc32e6654d4252b3ebbR22). lets see what team thinks about this.

Reply:
I like Occurrence. Thanks!